stable/prometheus-redis-exporter/README.md

```diff
@@ -116,3 +116,9 @@ data:
     return result
 ```
 If you want to use this script for collecting metrics, you could do this by just set `script.configmap` to the name of the configmap (e.g. `prometheus-redis-exporter-script`) and `script.keyname` to the configmap-key holding the script (eg. `script`). The required variables inside the container will be set automatically.
+
+## Upgrading
```

Comment:
Please also update the deployment and readme so that `.Values.redisAddress` is not a list anymore.

Reply:
done!